### PR TITLE
getting started v2

### PR DESCRIPTION
**Pull request by ChrisChinchilla**

### CODE REVIEW

**Review by thomasyit — APPROVED** [review]

File: docs/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
+# CAPE quickstart
+
+This guide shows you how to access and logging to your CAPE account portal and create the resources needed to run an application.
+
+!!! tip
+
+    This guide assumes you have received an email invitation from an account administrator.
+
+<!-- TODO: Add, cancel subscription etc, maybe break out? Seems an odd place to mention them and is it seamless anyway? -->
```
Comment (thomasyit) [inline docs/getting-started.md:9]: yup, will breakout, i will add a separate topic to it

File: docs/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+Create a self-hosted CAPE application instance from the _CAPE Application_ menu item and clicking the _Launch CAPE Application_ button.
+
+Clicking the button displays a command to copy and paste into a terminal. The command installs the CAPE application on the instance just created.
+
+<!-- TODO: More details on what script does -->
+
+After issuing the command, the CAPE application dashboard appears. You can login to the dashboard using the same credentials used to login to the portal.
+
+<!-- TODO: Can this not happen automatically without the need for a script? -->
```
Comment (thomasyit) [inline docs/getting-started.md:29]: @anufernando  any comments?

File: docs/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+<!-- TODO: Do you need to create a user to do anything more from here? -->
```
Comment (thomasyit) [inline docs/getting-started.md:31]: @anufernando same as above

File: docs/getting-started.md (hunk 2 above)
Comment (mooney-biqmind) [reply docs/getting-started.md:29]: @thomasyit for the own infrastructure version the user needs to run the script locally on their own cluster or execute the code/command given on their local cluster. Once this is done CAPE Web App will be installed locally on to their cluster. When they then navigate to the URL at the end of the script they will be prompted to log in and once successful they will access the dashboard.

File: docs/getting-started.md (hunk 3 above)
Comment (mooney-biqmind) [reply docs/getting-started.md:31]: @thomasyit the user they created the account with or a user they have added to the account (non billing) can be used to log in.

File: docs/getting-started.md (hunk 2 above)
Comment (ChrisChinchilla) [reply docs/getting-started.md:29]: @mooney-biqmind So this is only needed for self-hosted?

Comment (mooney-biqmind) [reply docs/getting-started.md:29]: @ChrisChinchilla Yes this only need for deploying CAPE app on to your own clusters.

**Pull request merged**